src/applications/gi/components/search/TypeOfInstitutionFilter.jsx

```diff
@@ -58,13 +47,35 @@ class TypeOfInstitutionFilter extends React.Component {
     }
 
     return (
-      <RadioButtons
-        label="Type of institution"
-        name="category"
-        options={options}
-        value={this.props.category}
-        onChange={this.props.onChange}
-      />
+      <div>
+        <RadioButtons
+          label="Type of institution"
+          name="category"
+          options={options}
+          value={this.props.category}
+          onChange={this.props.onChange}
+        />
+        {isVetTecSelected(this.props) && (
+          <div>
+            {/* full size */}
+            <span className="vads-u-margin-x--neg1  vads-u-display--none small-screen:vads-u-display--block">
+              <img
+                className="vettec-logo vads-u-padding-top--0p5 vads-u-margin-bottom--neg1"
+                src="/img/logo/vet-tec-logo.png"
+                alt="Vet Tec Logo"
+              />
+            </span>
+            {/* mobile size */}
+            <span className="vads-u-margin-bottom--1  vads-u-display--block small-screen:vads-u-display--none">
+              <img
+                className="vettec-mobile-logo"
```

Comment:
It looks like we've got a few things going on here:
1. We're rendering two `<span>` DOM nodes
2. The  `<span>`s have switching logic to determine which one is shown based on the viewport size
3. The `<span>`s also have differing margins
4. Inside the `<span>`s, we've got `<img>`s that are the same except the class names
5. These class names differ by controlling some margin and the size of the image

Does it maybe make more sense to do this all in SASS using screen size mixins and custom classes?

We'd then end up with something more like:
```jsx
{isVetTecSelected(this.props) &&
  <span className="vettec-logo-container">
    <img
      className="vettec-logo"
      src="/img/logo/vet-tec-logo.png"
      alt="Vet Tec Logo"
    />
  />
}
```

I'm not super keen on using JavaScript to solve a CSS problem, y'know?

Reply:
used sass logic to change height and width on landing page